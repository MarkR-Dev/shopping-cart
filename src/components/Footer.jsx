import styles from "../styles/Footer.module.css";

function Footer() {
  return (
    <>
      <footer className={styles.footer}>
        <div className={styles.container}>
          <div className={styles.footerLinks}>
            <ul>
              <li>About</li>
              <li>Store</li>
              <li>Team</li>
              <li>Blog</li>
            </ul>

            <ul>
              <li>Support</li>
              <li>FAQ</li>
              <li>Contact</li>
              <li>Jobs</li>
            </ul>

            <ul>
              <li>Legal</li>
              <li>Privacy</li>
              <li>Terms Of Service</li>
              <li>Refunds</li>
            </ul>

            <ul>
              <li>Socials</li>
              <li>Instagram</li>
              <li>Twitter</li>
              <li>GitHub</li>
            </ul>
          </div>
        </div>
      </footer>
    </>
  );
}

export default Footer;
