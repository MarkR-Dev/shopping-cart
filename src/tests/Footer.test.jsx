import { describe, it, expect } from "vitest";
import { render, screen } from "@testing-library/react";
import Footer from "../components/Footer";

describe("Footer Component", () => {
  it("Matches snapshot", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });

  it("Renders lists", () => {
    render(<Footer />);

    const lists = screen.getAllByRole("list");

    expect(lists.length).toBe(4);
  });
});
