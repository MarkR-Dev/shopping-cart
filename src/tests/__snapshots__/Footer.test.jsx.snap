// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer Component > Matches snapshot 1`] = `
<div>
  <footer
    class="_footer_4d8704"
  >
    <div
      class="_container_4d8704"
    >
      <div
        class="_footerLinks_4d8704"
      >
        <ul>
          <li>
            About
          </li>
          <li>
            Store
          </li>
          <li>
            Team
          </li>
          <li>
            Blog
          </li>
        </ul>
        <ul>
          <li>
            Support
          </li>
          <li>
            FAQ
          </li>
          <li>
            Contact
          </li>
          <li>
            Jobs
          </li>
        </ul>
        <ul>
          <li>
            Legal
          </li>
          <li>
            Privacy
          </li>
          <li>
            Terms Of Service
          </li>
          <li>
            Refunds
          </li>
        </ul>
        <ul>
          <li>
            Socials
          </li>
          <li>
            Instagram
          </li>
          <li>
            Twitter
          </li>
          <li>
            GitHub
          </li>
        </ul>
      </div>
    </div>
  </footer>
</div>
`;
